service/src/main/java/org/apache/celeborn/server/common/service/config/FsConfigServiceImpl.java

```diff
@@ -87,14 +87,14 @@ private Map<String, String> getConfigs(Map<String, Object> configMap) {
 
   private File getConfigFile(Map<String, String> env) throws IOException {
```

Comment:
What if we want set quota in a separated file?

Reply:
@AngersZhuuuu, why does the user set quota config in a separated file? IMO, the user always configures the dynamic config in one config file for unified management configuration.